airflow/security/permissions.py

```diff
@@ -16,25 +16,34 @@
 # under the License.
 
 # Resource Constants
-RESOURCE_DAGS = 'Dags'
+RESOURCE_ADMIN = 'Admin'
+RESOURCE_AIRFLOW = 'Airflow'
+RESOURCE_BROWSE_MENU = 'Browse'
+RESOURCE_DAGS = 'DAGs'
 RESOURCE_DAG_PREFIX = 'DAG:'
-RESOURCE_CONFIG = 'Config'
-RESOURCE_CONNECTION = 'Connection'
-RESOURCE_DAG_CODE = 'DagCode'
-RESOURCE_DAG_RUN = 'DagRun'
+RESOURCE_DOCS_MENU = 'Docs'
+RESOURCE_DOCS_LINK = 'Documentation'
+RESOURCE_CONFIG = 'Configurations'
+RESOURCE_CONNECTION = 'Connections'
+RESOURCE_DAG_CODE = 'DAG Code'
+RESOURCE_DAG_RUN = "DAG Runs"
 RESOURCE_IMPORT_ERROR = 'ImportError'
-RESOURCE_LOG = 'Log'
-RESOURCE_POOL = 'Pool'
-RESOURCE_TASK = 'Task'
-RESOURCE_TASK_INSTANCE = 'TaskInstance'
-RESOURCE_VARIABLE = "Variable"
+RESOURCE_JOB = "Jobs"
+RESOURCE_LOG = 'Logs'
```

Comment:
I wonder if for clarity we should

```suggestion
RESOURCE_AUDIT_LOG = 'Logs'
```

Reply:
Sure, I'm fine with that.